var/spack/repos/builtin/packages/pfunit/package.py

```diff
@@ -14,12 +14,13 @@ class Pfunit(CMakePackage):
 
     homepage = "http://pfunit.sourceforge.net/"
     url      = "https://github.com/Goddard-Fortran-Ecosystem/pFUnit/archive/3.2.9.tar.gz"
+    git      = "https://github.com/Goddard-Fortran-Ecosystem/pFUnit.git"
 
     maintainers = ['citibeth']
 
-    # Currently investigating build fails for v 4.0.0.
-    # See discussion in PR #11642.
-    # version('4.0.0',  sha256='b8b6470f2b1e2b19c164c244c10e803bd69c8da9a6a5a65ba7c479fb8b92a1e1') # noqa: E501
+    version('4.1.9',  tag='v4.1.9', submodules=True)
+    version('4.1.5',  tag='v4.1.5', submodules=True)
+    version('4.0.0',  tag='v4.0.0', submodules=True)
```

Comment:
Use the `/archive` mechanism to download tarballs as before, instead of git.  This URL works.  Note the 'v' has been added to the git tag starting with version 4.  That should now be the default; whereas earlier versions should get their own special URL.
```
curl -LO https://github.com/Goddard-Fortran-Ecosystem/pFUnit/archive/v4.1.9.tar.gz
```

Reply:
Thank you for the review!  Can you explain why the archive mechanism is preferred to git?